ert_shared/storage/oas.yml

```diff
@@ -0,0 +1,452 @@
+openapi: 3.0.1
+info:
+  title: ERT Storage API
+  description: API description in Markdown.
+  version: 0.1.0
+servers:
+- url: http://127.0.0.1:5000/
+paths:
+  /ensembles:
+    get:
+      summary: Returns a list of available ensembles.
+      description: Returns an overview of the ensembles available in the database.
+      responses:
+        200:
+          description: List of ensemble objects.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Ensemble-minimal'
+  /ensembles/{ensemble_id}:
+    get:
+      summary: Returns an ensemble.
+      description: Optional extended description in Markdown.
```

Comment:
maybe not keep placeholders ;)

Reply:
Damn it, though I found them all :sweat_smile: